app/javascript/dashboard/components/widgets/WootWriter/Editor.vue

```diff
@@ -23,6 +23,23 @@
       @change="onFileChange"
     />
     <div ref="editor" />
+    <div
+      v-show="isImageSelected && showImageToolbar"
+      class="image-toolbar absolute shadow-md rounded-[4px] flex gap-1 py-1 px-1 bg-slate-50 dark:bg-slate-700 text-slate-800 dark:text-slate-50"
```

Comment:
```suggestion
      class="absolute shadow-md rounded-[4px] flex gap-1 py-1 px-1 bg-slate-50 dark:bg-slate-700 text-slate-800 dark:text-slate-50"
```

Let's remove it if we are not using the class